src/game/default/g_ballistics.c

```diff
@@ -1215,17 +1218,29 @@ static void G_HookProjectile_Touch(g_entity_t *self, g_entity_t *other, const cm
 			gi.Multicast(self->s.origin, MULTICAST_PHS, NULL);
 		} else {
 
-			VectorNormalize(self->locals.velocity);
-			G_Damage(other, self, self->owner, self->locals.velocity, self->s.origin, vec3_origin, 5, 0, 0, MOD_HOOK);
-
 			gi.Sound(self, g_media.sounds.hook_gibhit, ATTEN_DEFAULT, (int8_t) (Randomc() * 4.0));
 
-			G_ClientHookDetach(self->owner);
-		}
+			if (g_hook_auto_refire->integer) {
+
+				G_ClientHookThink(self->owner, true);
+			} else {
+
+				VectorNormalize(self->locals.velocity);
+
+				G_Damage(other, self, self->owner, self->locals.velocity, self->s.origin, vec3_origin, 5, 0, 0, MOD_HOOK);
 
+				G_ClientHookDetach(self->owner);
+			}
+		}
```

Comment:
So is this fixed or fuxed?

Reply:
Fuxed. I'll comment it out when I get it ready to merge.